### DISCUSSION TITLE
How to generate static site with zero javascript with nuxt?

### DISCUSSION THREAD

**Discussion by itsmnthn** [nuxt/nuxt#27379] [General]
I have a static page which uses nuxt/seo , UnoCSS for styling and config file that has some data and links used throughout page and Components being reused and logically divided. I want to generate a static HTML page that do not use JS at all just pretender everything. Can also add more pages, I'm okay with page reload on page change.

Is this possible right now? how? if not what are the plans for this kind of behavior? can I write plugin or middleware that does this?

If anyone has done this please share, thanks.

**Comment by TheAlexLichter** [comment]
You can remove all JS via [the `features.noScripts` flag](https://nuxt.com/docs/guide/going-further/features#noscripts)
